game/hud/src/widgets/Compass/index.tsx

```diff
@@ -64,11 +79,20 @@ class Compass extends React.Component<CompassProps, CompassState> {
   }
 
   render() {
+    if (!this.state.visible) return null;
```

Comment:
I agree, but this is how all of the widgets are handling visibility at the moment. It's actually going to be useful, to make different parts of a widget visible or not.